### PR TITLE
Add chat room permission

### PR DESCRIPTION
**Pull request by pehala**
* Add permission checking to `Room`.
  * Permission is an optional field of  `Room` and only users with said permission can join the room.
* Depends on #46.

### CODE REVIEW

File: chat/models.py [hunk 1]
```diff
@@ -15,8 +15,9 @@ def get_prep_value(self, value):
 
 
 class Room(models.Model):
-    """Chat single_room"""
+    """Chat single room"""
     name = LowerCaseCharField(max_length=30, unique=True)
+    permission = models.CharField(max_length=30, default=None, blank=True, null=True)
```
Comment (Sanquira) [inline chat/models.py]: Is 30 enough?

File: chat/templates/chat/index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 {% block content %}
     <h1>Dostupné místnosti</h1>
-    {% for room in rooms %}
-        <a href="{% url 'chat:single_room' room_name=room.name %}">{{ room.name|title }}</a>
+    {% for room_info in rooms %}
+        <a href="{% url 'chat:single_room' room_name=room_info.name %}">{{ room_info.name|title }} ({{room_info.users}})</a>
```
Comment (Sanquira) [inline chat/templates/chat/index.html]: duplicity?? with other PR?

Comment (pehala) [reply chat/templates/chat/index.html]: The PR is dependant on the other one, as noted in the description

File: chat/models.py (hunk 1 above)
Comment (pehala) [reply chat/models.py]: Definitely for now, but that number is arbitrary a is there only because the CharField requires it.

**Pull request merged**